### DISCUSSION TITLE
EIP 1559 not supported

### DISCUSSION THREAD

**Discussion by mrsmkl** [0xPolygon/polygon-edge#340] [Ideas] [answered]
### Description

There's no EIP 1559 support yet.

### Your environment

* ubuntu 20.04
* 8c076ec050124689a0def6b0396e896632e35ef2
* develop

### Steps to reproduce

Run polygon sdk

For example
```
curl  http://localhost:8545 -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["0x1d",true],"id":12}'
```

### Expected behaviour

Return a block with field `baseFeePerGas`

### Actual behaviour

Returns a "legacy" block.

### Proposed solution

Hopefully support for eip 1559 is added soon...

**Comment by zivkovicmilos** [comment, marked as answer]
Hey @mrsmkl,

Thank you for reaching out.

Is support for EIP - 1559 something that's blocking you from using the Polygon SDK?

What project, if it's public, are you planning to integrate Polygon SDK with?

**Comment by mrsmkl** [comment]
We are using Polygon SDK for testing in https://github.com/nevermined-io and projects building on top of that. It won't block, I'll just implement a workaround to detect if the chain supports EIP-1559.
